example/eblob_backend.h

```diff
@@ -99,7 +99,11 @@ int blob_check_corrupted_stamp(void *buffer, size_t buffer_size);
 // 0 - data correct
 // -EILSEQ - data starts with stamp
 // error_code - an error has occurred
-int blob_read_and_check_stamp(const struct dnet_time *timestamp, int fd, uint64_t data_offset, uint64_t data_size);
+int blob_read_and_check_stamp(const struct eblob_backend_config *c,
```

Comment:
Don't forget to update comments for completeness.

Reply:
Thanks, fixed: 
https://github.com/interiorem/elliptics/pull/222/commits/45a2f9797bab5c9e227b6b62fcabb0872d174588